prisma/schema.prisma

```diff
@@ -19,41 +19,92 @@ model Account {
   accessTokenExpires DateTime? @map(name: "accessTokenExpires")
   createdAt          DateTime  @default(now()) @map(name: "createdAt")
   updatedAt          DateTime  @default(now()) @map(name: "updatedAt")
+
   @@index([providerAccountId], name: "providerAccountId")
   @@index([providerId], name: "providerId")
   @@index([userId], name: "userId")
   @@map(name: "accounts")
 }
 
 model Session {
-  id           Int      @default(autoincrement()) @id
+  id           Int      @id @default(autoincrement())
   userId       Int      @map(name: "userId")
   expires      DateTime
   sessionToken String   @unique @map(name: "sessionToken")
   accessToken  String   @unique @map(name: "accessToken")
   createdAt    DateTime @default(now()) @map(name: "createdAt")
   updatedAt    DateTime @default(now()) @map(name: "updatedAt")
+
   @@map(name: "sessions")
 }
 
 model User {
-  id            Int       @default(autoincrement()) @id
+  id            Int       @id @default(autoincrement())
   email         String?   @unique
   password      String?
   name          String?
   emailVerified DateTime? @map(name: "emailVerified")
   image         String?
   createdAt     DateTime  @default(now()) @map(name: "createdAt")
   updatedAt     DateTime  @default(now()) @map(name: "updatedAt")
+
+  submissions Submission[]
   @@map(name: "users")
 }
 
 model VerificationRequest {
-  id         Int      @default(autoincrement()) @id
+  id         Int      @id @default(autoincrement())
   identifier String
   token      String   @unique
   expires    DateTime
   createdAt  DateTime @default(now()) @map(name: "createdAt")
   updatedAt  DateTime @default(now()) @map(name: "updatedAt")
+
   @@map(name: "verificationRequests")
-}
\ No newline at end of file
+}
+
+model PuzzleType {
+  id        Int      @id @default(autoincrement())
+  name      String   @unique @db.VarChar(255)
+  createdAt DateTime @default(now()) @map(name: "created_at")
+  updatedAt DateTime @updatedAt @map(name: "updated_at")
+
```

Comment:
No need space here(?)

Reply:
This happens a few more times too, I'll fix them all